recipes/ecell4_base/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "ecell4_base" %}
+{% set version = "2.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/ecell/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: 6342531d37f5e239a037c2cee13a580752b77b10f72858bd870a641a675aa3c3
+    patches:
+      - 3713c26ca11e60b1d429a624df26e8277be6f634.patch
+  - url: https://github.com/pybind/pybind11/archive/v2.4.3.tar.gz
+    sha256: 1eed57bc6863190e35637290f97a20c81cfe4d9090ac0a24f3bbf08f265eb71d
+    folder: pybind11
+  - url: https://github.com/ecell/greens_functions/archive/v{{ version }}.tar.gz
+    sha256: f20875de65e611351acc0ab35c2e2302648ed501f87f4f6471be747ffd3b32ed
+    folder: greens_functions
```

Comment:
Also it seems as though this should be a separate package as well.

Reply:
This `greens_functions` is a pure C++ project, and only for `ecell4_base`.
So, we want to archive its sources into a tarball of `ecell4_base` instead of publishing it as another package. Is it OK?

Reply:
@scopatz We removed pybind11 from meta.yaml `source:` and included greens_functions in the release of ecell4_base.